redaxo/src/core/lib/packages/manager.php

```diff
@@ -3,14 +3,16 @@
 /**
  * Manager class for packages.
  *
+ * @template T as rex_package
+ *
  * @package redaxo\core\packages
  */
 abstract class rex_package_manager
 {
     use rex_factory_trait;
 
     /**
-     * @var rex_package
+     * @var T
```

Comment:
Besser mit psalm-var arbeiten, sonst geht ja die autovervollständigung in phpstorm hier kaputt (vermutlich).